source/codegen/service.cpp.mako

```diff
@@ -289,6 +289,8 @@ one_of_case_prefix = f'{namespace_prefix}{function_name}Request::{PascalFieldNam
     % elif c_type == 'ViSession':
       auto ${parameter_name}_grpc_session = ${request_snippet};
       ${c_type} ${parameter_name} = session_repository_->access_session(${parameter_name}_grpc_session.id(), ${parameter_name}_grpc_session.name());\
+    % elif c_type == 'ViInt32[]':
```

Comment:
It can be added that way but it simply introduces more level of branching. We can keep this comment open and if owners also ask for that, I can change it.